source/state_representation/tests/testJointState.cpp

```diff
@@ -14,7 +14,8 @@ TEST(SetPositonsWithModulo, PositiveNos) {
 
   std::cerr << j1 << std::endl;
 
-  for (unsigned int i = 0; i < j1.get_size(); ++i) EXPECT_TRUE(-M_PI < j1.get_positions()(i) && j1.get_positions()(i) < M_PI);
+  for (unsigned int i = 0; i < j1.get_size(); ++i)
+    EXPECT_TRUE(-M_PI < j1.get_positions()(i) && j1.get_positions()(i) < M_PI);
```

Comment:
Not in the scope of this PR but if the intention is to remove joint boundaries from JointPosition then this test case should be the first thing changed.

Reply:
Definitely. This is my next PR